### PR TITLE
feat: [CCM-6959]: GCP UI enhancements

### PR DESCRIPTION
**Pull request by rishik30**
##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by rishik30** [comment]
trigger cypress

**Review by rajarshee-harness — DISMISSED** [review]

File: src/modules/75-ce/components/COGatewayConfig/steps/ManageResources/ManageResources.tsx [hunk 1]
```diff
@@ -54,8 +55,13 @@ const managedResources = [
   {
     label: 'VM(s)',
     value: RESOURCES.INSTANCES,
-    providers: ['azure', 'gcp'],
-    ffDependencies: [null, FeatureFlag.CE_AS_GCP_VM_SUPPORT]
+    providers: ['azure']
+  },
+  {
+    label: 'GCE VM(s)',
```
Comment (jenil-harness) [inline src/modules/75-ce/components/COGatewayConfig/steps/ManageResources/ManageResources.tsx:61]: move all the labels to strings.yml

**Review by jenil-harness — CHANGES_REQUESTED** [review]

File: src/modules/75-ce/components/COGatewayConfig/steps/ManageResources/ManageResources.tsx (hunk 1 above)
Comment (jenil-harness) [reply src/modules/75-ce/components/COGatewayConfig/steps/ManageResources/ManageResources.tsx:61]: You can pass `getString` function to it. or return string id

**Review by jenil-harness — DISMISSED** [review]

**Review by rajarshee-harness — APPROVED** [review]

File: src/modules/75-ce/components/COGatewayConfig/steps/ResourceFulfilment.tsx [hunk 2]
```diff
@@ -142,38 +145,28 @@ const ResourceFulfilment: React.FC<ResourceFulfilmentProps> = props => {
     [props.gatewayDetails.routing.container_svc]
   )
 
-  const getTitle = () => {
-    let title = getString('ce.co.autoStoppingRule.configuration.step3.title')
-    if (props.selectedResource === RESOURCES.ASG) {
-      title = getString('ce.co.autoStoppingRule.configuration.step3.asgTitle')
+  const shouldHide = () => {
+    let flag
```
Comment (jyotiarora-harness) [inline src/modules/75-ce/components/COGatewayConfig/steps/ResourceFulfilment.tsx:149]: By default we can keep this flag as false and combine the below cases with OR

Comment (rishik30) [reply src/modules/75-ce/components/COGatewayConfig/steps/ResourceFulfilment.tsx:149]: that was the approach earlier but sonar created an issue for that so boiled down to this.

Comment (jyotiarora-harness) [reply src/modules/75-ce/components/COGatewayConfig/steps/ResourceFulfilment.tsx:149]: ok

**Pull request merged**